<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DuAltaLibroForm</class>
 <widget class="QWidget" name="DuAltaLibroForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Libro</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="nombreLabel">
       <property name="text">
        <string>Nombre</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nombreLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="autorLabel">
       <property name="text">
        <string>Autor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="autorLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="autorSeleccionarPushButton">
       <property name="text">
        <string>Seleccionar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="editorialLabel">
       <property name="text">
        <string>Editorial</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editorialLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editorialSeleccionarPushButton">
       <property name="text">
        <string>Seleccionar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
